packages/prosess-uttak-antall-dager-sluttfase/src/AntallDagerLivetsSluttfaseIndex.tsx

```diff
@@ -25,51 +25,54 @@ interface OwnProps {
 const AntallDagerLivetsSluttfaseIndex = ({
   kvoteInfo
 }: OwnProps) => {
-  if(!kvoteInfo){
+  if (!kvoteInfo) {
     return null;
   }
 
   const maxAntallDager = 60;
-  const antallDagerGjenstår = maxAntallDager - (kvoteInfo.forbruktKvoteHittil + kvoteInfo.forbruktKvoteDenneBehandlingen);
+  const antallDagerGjenstar = maxAntallDager - (kvoteInfo.totaltForbruktKvote);
+
+  const gronneDager = () => {
+    if (kvoteInfo.totaltForbruktKvote <= maxAntallDager) return kvoteInfo.totaltForbruktKvote;
```

Comment:
Kan ikke detta vara en variabel? Så att man ikke har funktioner for noe som utan parameter og statisk.

Endre variabelnavn till noe mer forklarende?

Reply:
Exempelvis:

```
  const antallDagerBruktInnenforKvoten = kvoteInfo.totaltForbruktKvote <= maxAntallDager ? kvoteInfo.totaltForbruktKvote : 0;
  const antallDagerNårKvotenHarBlivitBruktOver = kvoteInfo.totaltForbruktKvote > maxAntallDager ? maxAntallDager : 0
```;

Hvis det er passande variabelnavn, jeg antar att det er hva som skjer utifrån koden?